### PR TITLE
add appropriate logging, move message processing into index.js

### PR DESCRIPTION
**Pull request by costajohnt**

### CODE REVIEW

File: src/VimeoUploadLambda/index.js [hunk 1]
```diff
@@ -45,17 +46,30 @@ const updateAsset = (message, vimeoId) => {
...
     if (err) throw err;
+    console.log(`updated ${result.affectedRows} rows`);
   });
 
   connection.end();
 };
 
+const processSNS = event => {
+  let message = event.Records[0].Sns.Message;
+  message = JSON.parse(message);
```
Comment (kofno) [inline src/VimeoUploadLambda/index.js]: why this extra assignment? Couldn;t this just be:

```
const message = event.Records[0].Sns.Message;
return JSON.parse(message);
```

**Review by kofno — COMMENTED** [review]
Serverless should let us use ES6 and such, right?

**Comment by ecopony** [comment]
Please follow the same commit-message standards that we have for our core repo.

**Comment by costajohnt** [comment]
@ecopony @kofno comments addressed.  how we lookin?

**Pull request merged**